class/practice2.1/NegativeMax.cpp

```diff
@@ -0,0 +1,41 @@
+#include <algorithm>
```

Comment:
This file was not expected to be a part of the task while you can see one of the files attached to the task related to implementation with ranges libraries if you want to learn more about modern c++